draft-looker-oauth-attested-key-based-client-authentication.md

```diff
@@ -196,9 +209,9 @@ The following example is the decoded header and payload of a JWT meeting the pro
 }
 ~~~
 
-### Client Key Attestation PoP JWT {#client-key-attestation-pop-jwt}
+### Client Attestation PoP JWT {#client-attestation-pop-jwt}
 
-The following rules apply to validating the client key attestation PoP JWT. Application of additional restrictions and policy are at the discretion of the authorization server.
+The following rules apply to validating the client attestation PoP JWT. Application of additional restrictions and policy are at the discretion of the authorization server.
```

Comment:
```suggestion
The following rules apply to validating the Client Attestation JWT. Application of additional restrictions and policy are at the discretion of the authorization server.
```
Client Attestation JWT or Client Attestation PoP JWT?
also capitalization - please be consistent...